openedx/openedx-event-sink-clickhouse
# fix: sink ccx courses on original course published

**Description:** This PR sinks the CCX child courses when the master course is published.

**Testing instructions:**

1. Enable CCX courses
2. Create a CCX course
3. Publish the master course
4. Verify that the CCX child courses are sink

**Author concerns**

Although there is a signal receiver for the `course_published` signal in the CCX Django application, this application has been written for the LMS, so the signal is never connected. Refactoring it would require a major development effort that's out of scope for this project. Also, doing that refactoring will not be compatible with older `edx-platform` releases.
